.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  'env': {
+    'browser': true,
+    'node': true
+  },
+  'globals': {
+    'window': true,
+    'document': true,
+    'jQuery': true,
+    '$': true,
+    'angular': true
```

Comment:
Yes....? I don't know, we probably could but no one on the project really knows angular well enough to implement at this time. :'( @bbrock25 can testify.